features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx

```diff
@@ -0,0 +1,229 @@
+    },
+    stepper: {
+        background: theme.palette.background.default,
+    },
+    box: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        background: 'white',
+        padding: '2px',
+        width: '405px',
+        height: '82px',
+        borderRight: '0',
+        marginRight: '0',
+
+
+    },
+}));
+
+/**
+ * This component renders the requirements list
+ * @param {*} props
+ */
+export default function CustomizedSteppers(props) {
+    const restApi = new Api();
+    const { api } = props;
+    const classes = useStyles();
+    const isEndpointAvailable = api.endpointConfig !== null;
+    const isTierAvailable = api.policies.length !== 0;
+    const isPrototypedAvailable =
+        api.endpointConfig !== null && api.endpointConfig.implementation_status === 'prototyped';
+
+    const [lifecycleState, setlifecycleState] = useState([]);
+
+
+    useEffect(() => {
+        restApi.getLcState(api.id)
+            .then((result) => {
+                setlifecycleState(result.body.state);
+            });
+    }, []);
+
+    let activeStep = 0;
+    if (lifecycleState === 'Created' && ((isEndpointAvailable && isTierAvailable) || isPrototypedAvailable)) {
+        activeStep = 2;
+    } else if (lifecycleState === 'Created') {
+        activeStep = 1;
+    } else if (lifecycleState !== 'Created') {
+        activeStep = 3;
+    }
+    return (
+        <div className={classes.root}>
+            <Stepper alternativeLabel activeStep={activeStep} className={classes.stepper}>
+                <Step className={classes.label}>
+                    <StepLabel style={{ position: 'relative' }}>
+                        <Box className={classes.box}>
+                            <Typography variant='h6'>
+                                Created
+                            </Typography>
+                        </Box>
+                        <ArrowForwardIosRoundedIcon className={classes.arrowIcon} />
+                    </StepLabel>
+                </Step>
+                <Step className={classes.label}>
+                    <StepLabel style={{ position: 'relative' }} >
+                        <Box p={2} bgcolor='white' width='377px' borderColor='grey.500' borderLeft='0' borderRight='0' >
+                            <Typography variant='h10'>
+                            Requirements
+                            </Typography>
+                            <Grid xs={12} className={classes.gridRequirements}>
+                                <Grid xs={4} className={classes.gridSmall}>
+                                    {isEndpointAvailable ? (
+                                        <CheckIcon className={classes.iconTrue} />
+                                    ) : (
+                                        <CloseIcon className={classes.iconFalse} />
+                                    )}
+
+                                    <Typography variant='caption'>Endpoint</Typography>
```

Comment:
Should be Endpoints. Add i18n as well

Reply:
It should be Endpoint. In order to satisfy the basic requirement you only need to provide one endpoint.
Added i18n.